components/motor/gpio/motor_encoder_test.go

```diff
@@ -744,7 +742,7 @@ func TestDirFlipMotor(t *testing.T) {
 	defer enc.Close(context.Background())
 
 	enc.AttachDirectionalAwareness(&fakeDirectionAware{m: dirflipFakeMotor})
-	dirFMotor, err := NewEncodedMotor(resource.Config{}, cfg, dirflipFakeMotor, e, logger)
+	dirFMotor, err := WrapMotorWithEncoder(context.Background(), e, resource.Config{}, cfg, dirflipFakeMotor, logger)
```

Comment:
I have a slight preference for lines that aren't too long, though the definition of "too long" changes from person to person, depending in part on how big their screen is and what size font they use. If these lines are a typical length for this file, that's fine. but if you've added the longest lines in here, I'd prefer for them to be line-wrapped (perhaps put a newline just before the context?).

Reply:
just skimmed the file and these are on the longer end, but still pretty typical for this file, so I'm going to leave as is because I think it's a little easier to read this way